src/common/processing_linux.c

```diff
@@ -11,7 +11,7 @@
 #include <errno.h>
 #include <sys/wait.h>
 
-enum { FF_PIPE_BUFSIZ = 4096 };
+#define FF_PIPE_BUFSIZ 16384
```

Comment:
You once again assume unreliable things. The pipe buf size has been increased twice, how can you be sure that it won't be increased again?

However, I think it should be ok even if FF_PIPE_BUFSIZ is smaller than real pipe buf size.

Reply:
> You once again assume unreliable things. The pipe buf size has been increased twice, how can you be sure that it won't be increased again?
> 
> However, I think it should be ok even if FF_PIPE_BUFSIZ is smaller than real pipe buf size.

It's not my assume, 16384 is the default size of macOS/BSDs pipe buffer.

If this confuse you, i'll change it back to 4096.

Reply:
Fixed.